doc/api/process.markdown

```diff
@@ -116,6 +116,45 @@ Nine out of ten times nothing happens - but the 10th time, your system is bust.
 
 You have been warned.
 
+## Event: 'unhandledRejection'
+
+Emitted whenever a `Promise` is rejected and no error handler is attached to the promise within a turn of the event loop. When programming with promises exceptions are encapsulated as rejected promises. Such promises can be caught and handled using `promise.catch(...)` and rejections are propagated through a promise chain. This event is useful for detecting and keeping track of promises that were rejected whose rejections were not handled yet. This event is emitted with the following arguments:
+
+ - `reason` the rejection reason of the promise susprected in having an unhandled rejection
```

Comment:
susprected/suspected. But also I think this is better rephrased as simply something like "the object with which the promise was rejected (usually an `Error` instance)"